### PR TITLE
Create a new component for owner-avatar

### PR DESCRIPTION
**Pull request by JoshK2**
## Proposed Changes

- Create a new component for the owner avatar that use the UI avatar.
- Remove unnecessary code from the avatar component that was used for the owner.

### CODE REVIEW

**Review by odedre — APPROVED** [review]

**Review by odedre — CHANGES_REQUESTED** [review]
see changes

**Review by odedre — APPROVED** [review]

File: scopes/design/ui/owner-avatar/owner-avatar.tsx [hunk 1]
```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import classNames from 'classnames';
+import { UserAvatar, UserAvatarProps } from '@teambit/ui.avatar';
+import styles from './owner-avatar.module.scss';
+
+export type OwnerAvatarProps = {
+  avatarProps: UserAvatarProps;
+} & React.HTMLAttributes<HTMLElement>;
```
[the remark this answers is not in the document]
Comment (JoshK2) [reply scopes/design/ui/owner-avatar/owner-avatar.tsx]: done

File: scopes/design/ui/owner-avatar/owner-avatar.tsx [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+export function OwnerAvatar({ avatarProps, className }: OwnerAvatarProps) {
+  return (
+    <div className={classNames(styles.isOwner, className)}>
```
[the remark this answers is not in the document]
Comment (JoshK2) [reply scopes/design/ui/owner-avatar/owner-avatar.tsx]: done

File: scopes/design/ui/owner-avatar/owner-avatar.tsx [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+      <UserAvatar {...avatarProps} />
```
[the remark this answers is not in the document]
Comment (JoshK2) [reply scopes/design/ui/owner-avatar/owner-avatar.tsx]: done

File: scopes/design/ui/owner-avatar/owner-avatar.docs.md [hunk 4]
```diff
@@ -0,0 +1,20 @@
+---
+labels: ['react', 'typescript', 'avatar', 'owner', 'ui']
+description: 'An owner avatar component'
+---
+
+import { OwnerAvatar } from './owner-avatar';
+
+An owner avatar component that envelope the Avatar component.
```
[the remark this answers is not in the document]
Comment (JoshK2) [reply scopes/design/ui/owner-avatar/owner-avatar.docs.md]: done

**Pull request merged**